fcd_sale_order/models/stock_move_line.py

```diff
@@ -43,6 +43,8 @@ class StockMoveLine(models.Model):
         currency_field="sale_currency_id",
         store=True,
     )
+    fcd_document_id = fields.Many2one(related='fcd_document_line_id.fcd_document_id', store=True)
```

Comment:
Or it's stored and then we should add its initialization in pre-migration script, or make it non-stored. If non-stored, is it available in Transfers XLSX export templates? If available, I'd make it non-stored and check speed in Test environment